src/containers/Sidebar.js

```diff
@@ -62,7 +62,7 @@ export class Sidebar extends Component {
 
     return (
       <div>
-        {collectionItems.length > 0 && (
+        {collectionItems.length && (
```

Comment:
This could render `0` if there are no collections. Could you add a test for that case?

Reply:
Isn't `0` a falsey value in JS?
There's already a similar use in the component at `master`:
https://github.com/jekyll/jekyll-admin/blob/167b2bf3b7036aef3f88a27b44331f051ef1dbb0/src/containers/Sidebar.js#L22-L24


> Could you add a test for that case?

There's already a test for *rendering Sidebar successfully with zero collections* with a defined snapshot.
Since there's no change in the snapshot at this commit, is this still a blocker?

Reply:
These are different cases. In this example, it returns `null` which renders nothing. But I wasn't sure `collectionItems.length` is converted to `falsy` value. If snapshot is the same, it works as expected then 👍

Reply:
You were right @mertkahyaoglu. This does indeed render `0`. I tested manually.